bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -57,22 +57,7 @@
             "displayName": "KIE Admin Password",
             "description": "KIE administrator password",
             "name": "KIE_ADMIN_PWD",
-            "from": "[a-zA-Z]{6}[0-9]{1}!",
-            "generate": "expression",
-            "required": false
-        },
-        {
-            "displayName": "KIE Server Controller User",
-            "description": "KIE server controller username (adds the user)",
-            "name": "KIE_SERVER_CONTROLLER_USER",
-            "value": "controllerUser",
-            "required": false
-        },
-        {
-            "displayName": "KIE Server Controller Password",
-            "description": "KIE server controller password (adds the user)",
-            "name": "KIE_SERVER_CONTROLLER_PWD",
```

Comment:
Removed because it was duplicated, so we assure password matches in both BC (which creates it) and Router (which uses it)